public/api/swagger.json

```diff
@@ -0,0 +1,838 @@
+{
+    "swagger": "2.0",
```

Comment:
can we nest these with two spaces instead of 4? Finding the deep nesting hard to read..